services/s3/src/test/java/software/amazon/awssdk/services/s3/functionaltests/MultiRegionAccessPointChecksumTest.java

```diff
@@ -190,6 +190,24 @@ public void syncStreamingInput_checksumCalculation(RequestChecksumCalculation re
         }
     }
 
+    @ParameterizedTest(name = "{index} {3}")
+    @MethodSource("streamingInputChecksumCalculationParams")
+    public void asyncStreamingInput_checksumCalculation(RequestChecksumCalculation requestChecksumCalculation,
```

Comment:
Can we add an integ test as well?

Reply:
The regression test suite has been updated to test MRAP buckets with payload signing on/off.